bash_completion.sh

```diff
@@ -269,6 +272,19 @@ _buildtest ()
             return
           esac
         ;;
+        profile)
+          local opts="--help -h list"
+          COMPREPLY=( $( compgen -W "${opts}" -- $cur ) )
+          if [[ "${prev}" == "list" ]]; then
+            local opts="--help  --theme --yaml -h -t -y"
+            COMPREPLY=( $( compgen -W "${opts}" -- $cur ) )
+          fi
+
+          case "${prev}" in --theme|-t)
```

Comment:
can you remove the `-t` option this will be removed in https://github.com/buildtesters/buildtest/pull/1576